### ISSUE TITLE
Nothing shown when checking myip

### ISSUE THREAD

**Issue by sherlock-ma** [SukkaW/zsh-proxy#6]
Hi when I type `myip` no meaningful information is shown. What I get is
```
========================================
Check what your IP is
----------------------------------------
IPv4: ----------------------------------------
IPv6: 
----------------------------------------
Info: 
No JSON object could be decoded

========================================
```

What could be the issue? Does it mean that my proxy is not working properly?

**Comment by SukkaW** [comment]
@sherlock-ma Did you try upgrading the plugin to the latest?

**Comment by sherlock-ma** [comment]
Yes I did. I just used `antigen update` to update it but it still does not work. Specifically I am using SOCKS5 with the default setting `127.0.0.1:1080`. The SOCKS5 proxy should be working correctly because I am using `v2ray`, with SOCKS5 being the first hop.

**Comment by JingxinLee** [comment]
Same issue.

**Comment by sylph520** [comment]
same issue.

**Comment by Liuxw17** [comment]
Enable proxy for:
- shell
- git
/usr/bin/env: ‘bash\r’: No such file or directory
/usr/bin/env: ‘bash\r’: No such file or directory
- npm
- apt
Done!
========================================
Check what your IP is
----------------------------------------
